src/components/common/services/index.tsx

```diff
@@ -27,6 +33,15 @@ const useStyles = makeStyles({
   },
 });
 
+const iconComponents: { [key: string]: React.ReactElement } = {
+  'webDev-icon': <WebDevIcon />,
```

Comment:
pls use camelcase as key name
```suggestion
  'webDevIcon': <WebDevIcon />,
```